test/suites/projects.sh

```diff
@@ -1060,3 +1060,31 @@ test_projects_usage() {
   lxc image delete testimage --project test-usage
   lxc project delete test-usage
 }
+
+test_projects_yaml() {
+  lxc project create test-project-yaml < <(cat <<EOF
```

Comment:
Is the process substitution intentional here, or could we remove it to for readability? 
Something like that:
```sh
lxc project create test-project-yaml <<EOF
config:
  limits.cpu: 2
  limits.memory: 2048MiB
description: Test project using YAML
EOF
```

Reply:
fixed